public_html/admin/controller/responses/sale/contact.php

```diff
@@ -126,9 +139,9 @@ public function abort()
         //init controller data
         $this->extensions->hk_InitData($this, __FUNCTION__);
 
-        $task_id = (int)$this->request->post['task_id'];
+        $task_id = (int) $this->request->post['task_id'];
         if (!$task_id) {
-            return null;
+            return;
```

Comment:
Is this a new standard? Before PHP required something in return

Reply:
no. it's old feature for "void" methods